src/app/shared/models/models.component.ts

```diff
@@ -284,16 +319,21 @@ export class ModelsComponent implements OnInit, OnDestroy {
     // this.stateHandler.NewWindow('modelscomparison', { sourceId: source.id, targetId: target ? target.id : null });
   };
 
-  loadModelsToCompare = dataModel => {
-    this.resources.catalogueItem.listSemanticLinks(dataModel.domainType, dataModel.id, { all: true }).subscribe(result => {
-      const compareToList = [];
-      const semanticLinks = result.body;
-      semanticLinks.items.forEach(link => {
-        if (['Superseded By', 'New Version Of'].indexOf(link.linkType) !== -1 && link.source.id === dataModel.id) {
-          compareToList.push(link.target);
-        }
+  loadModelsToCompare = (dataModel) => {
```

Comment:
If in this area, might as well change this to `loadModelsToCompare(dataModel) { }` instead